opencensus/stats/internal/measure.cc

```diff
@@ -14,12 +14,22 @@
 
 #include "opencensus/stats/measure.h"
 
+#include "absl/strings/string_view.h"
 #include "opencensus/stats/internal/measure_registry_impl.h"
 #include "opencensus/stats/measure_registry.h"
 
 namespace opencensus {
 namespace stats {
 
+// static
+template <typename MeasureT>
+Measure<MeasureT> Measure<MeasureT>::Register(absl::string_view name,
```

Comment:
I'm sorry to be a pain but since this is already an API change, this would be a great time to go from (name,units,description) to (name,description,units) to match the argument order in opencensus-java and opencensus-go.

Reply:
Done. (And MeasureInt->MeasureInt64, which I think is less important but still nice while we are breaking everything.)